styles.css

```diff
@@ -0,0 +1,59 @@
+* {
+  background-color: #FFF;
+  margin: 0;
+  padding: 0;
+}
+
+.nav-list {
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+  list-style: none;
+  padding: 1rem;
+  margin: 1rem;
+}
+
+#nav-bar-btn {
+  border: none;
+}
+
+#nav-bar-p {
+  font-family: 'Inter';
+  font-weight: 600;
+  font-size: 1.7rem;
+  color: #28352F;
+}
+
+.heading-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-around;
+  text-align: center;
+  font-family: 'Inter';
+  font-style: normal;
+  height: 750px;
+  background-image: url('images/header-illsutration-mobile@2x.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+#main-heading {
+  font-weight: bold;
+  font-size: 3.4rem;
+  color: #172B4D;
+}
```

Comment:
- [ ] Kindly make sure the font size and font style of your h1 tag match with Figma designs 

figma:
![image](https://user-images.githubusercontent.com/43564324/164347042-35dd77d2-a76e-4798-a62b-d96956756f24.png)

your design:
![image](https://user-images.githubusercontent.com/43564324/164347398-b2a253a4-22a1-4d10-88d7-a5c3591fb287.png)